src/backend/prisma/manual.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
```

Comment:
my only suggestion would be putting some kind of documentation/comment here about how this is only for devops stuff blah blah blah and maybe explain it a bit yk